package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "devDependencies": {
-    "@percy/agent": "^0.1.14",
+    "@percy/agent": "~0",
```

Comment:
hmm

Reply:
Sorry, this was me playing with GH's UI.

Reply:
AH! That makes sense 😆